c31-swynlake/reactive-nutshell
# Git i tdone

# Description

Added friends list to app views state. Added new section of database for connections for friends. 

Applies to issue # 8

## Type of change

Please delete options that are not relevant.



# Testing Instructions

1. git fetch gitITdone
1. copy example file to JSON. 
1. login and check friends in state, also register new user and check database to see that new friend ID is created. reload page and make sure active user is still logged in. 



# Checklist:
